components/twilio/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/twilio",
-  "version": "0.3.11",
+  "version": "0.3.12",
```

We should bump a minor version since there are new features (components)
```suggestion
  "version": "0.4.0",
```